### DISCUSSION TITLE
Trimble GNSS receivers and QField

### DISCUSSION THREAD

**Discussion by dukytony** [opengisch/QField#3837] [Q&A]
Hi everyone,

I would need some help for a surveying setup : 

I have 1 trimble R8s (base), 1 trimble R2(mobile), 1 trimble R6(mobile), 1 trimble R4(base)

I'm using a base-mobile setup with corrections sent from the base to the mobile using radio.

I would like to be able to receive NMEA stream from my mobile rover to Qfield running on an android samsung tablet so that I could record new point with already corrected gnss positions. (Fix)

My tablet needs to stay connected to 4G network because I'm accessing remote postgis tables for my survey.

What's the best way to do this? serial port? tcp? udp? (will tcp or udp work with the 4g connected to the internet?)

Please help!

Thanks a lot!

**Comment by lucienicolier** [comment]
Hi,

You will find the doc about GNSS positioning here [Positioning GNSS](https://docs.qfield.org/how-to/gnss/?h=gnss)

Hope you will find all your answers.

Best regards
